lms/resources/lti_launch.py

```diff
@@ -106,6 +106,36 @@ def h_groupid(self):
         """
         return f"group:{self.h_authority_provided_id}@{self._authority}"
 
+    def h_section_groupid(self, tool_consumer_instance_guid, context_id, section_id):
```

Comment:
`tool_consumer_instance_guid` and `context_id` can of course be retrieved from `self.request.params` but they won't be able to be when this function is actually called--it's going to be called by an API view not an LTI launch request and `self.request.params` won't contain all the launch params. Plus it's kind of better to pass them in as arguments anyway.

Many of the other methods in this class are soon going to need to be refactored similarly